linq4j/src/main/java/org/apache/calcite/linq4j/tree/Primitive.java

```diff
@@ -384,9 +384,15 @@ static void checkRoundedRange(Number value, double min, double max) {
     }
   }
 
+  /** Called from BuiltInMethod.INTEGER_CAST */
```

Comment:
javadoc is same as next method

if methods have the same description, are both required?